python-packages/smithy-http/BUILD

```diff
@@ -0,0 +1,31 @@
+resource(name="pyproject", source="pyproject.toml")
+resource(name="readme", source="README.md")
+resource(name="notice", source="NOTICE")
+
+python_distribution(
+    name="dist",
+    dependencies=[
+        ":pyproject",
+        ":readme",
+        ":notice",
+        "python-packages/smithy-http/smithy_http:source",
+        "python-packages/smithy-core:dist",
+    ],
+    provides=python_artifact(
+        name="smithy_http",
+        version="0.0.1",
+    ),
+)
+
+# We shouldn't need this, but pants will assume that smithy-core is an external
+# dependency since it's in pyproject.toml and there's no way to exclude it, so
+# for now we need to duplicate things.
+python_requirements(
+    name="requirements",
+    source="requirements.txt",
+)
+
+# python_requirements(
+#     name="pyproject-requirements",
+#     source="pyproject.toml",
+# )
```

Comment:
Did we mean to delete this or was there a reason to keep it commented out?

Reply:
Ah yeah, it needs to be removed. Essentially there was a problem where including the local dependencies in the pyproject would cause pants to fail because it tries to find it as an external dependency. Pants doesn't currently have the capability to make additions to pyproject files unfortunately, only `setup.py`